docs/Toml Help.md

```diff
@@ -0,0 +1,341 @@
+**Note:** A high-level understanding of TOML is recommended. For more information, see [TOML](https://github.com/toml-lang/toml).
+
+While the pipeline entry point is defined in the `Jenkinsfile`, the actual substance of a VeriStand custom device build is defined in `build.toml`. This document describes the available configuration options in `build.toml` and how those options impact the actual build.
+
+There are two types of configuration captured in the build file:
+1. Non-stage configuration
+2. Stage configuration
+
+**Table of Contents**
+* [Non-Stage Configuration](https://github.com/ni-veristand-cds/commonbuild/wiki/TOML-for-ni-veristand-cds#non-stage-configuration)
```

Comment:
All items in the table of contents still point to the wiki location. These need to be updated to just use the [anchors](https://gist.github.com/asabaylus/3071099).

Reply:
Good catch. Also, because of this comment I noticed that there were some links to an older repository, so I changed those to this one via anchors.